src/main/java/com/example/book/store/config/SecurityConfig.java

```diff
@@ -0,0 +1,41 @@
+package com.example.book.store.config;
+
+import static org.springframework.security.config.Customizer.withDefaults;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.web.SecurityFilterChain;
+
+@RequiredArgsConstructor
+@Configuration
+public class SecurityConfig {
+    private final UserDetailsService userDetailsService;
+
+    @Bean
+    public PasswordEncoder getPasswordEncoder() {
+        return new BCryptPasswordEncoder();
+    }
+
+    @Bean
+    public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
+        return http
+                   .cors(AbstractHttpConfigurer::disable)
+                   .csrf(AbstractHttpConfigurer::disable)
+                   .authorizeHttpRequests(
+                       auth -> auth
+                                   .requestMatchers("/api/auth/registration")
```

Comment:
.requestMatchers("/auth/**", "/swagger-ui/**", "/v3/api-docs/**")

we shouldn't filter auth/login too